implementation/leprechaun/src/test-setup.ts

```diff
@@ -0,0 +1,12 @@
+import { MetadataStorage, MikroORM } from '@mikro-orm/core';
+import { testConfig } from './config/mikro-orm.test.config';
+
+let _database: MikroORM;
+
+export default async (globalConfig, projectConfig) => {
+  const database = await MikroORM.init(testConfig);
+  await database.schema.createSchema();
+  globalThis.database = database;
```

Comment:
I hate how we're using global state for this, but I believe that you made sure this is necessary.

Reply:
I need this to create schema for testing and tear it down afterwards. `Jest` fucks with something in the background, and I can't just recreate connection from scratch and drop schema. I did not debug much, but it is the only way that I know works.